File: logica-programacao/javascript/034.js
/* Peça duas notas.
A pessoa é aprovada se:

média ≥ 7
ou

média ≥ 6 E nenhuma nota é menor que 5

Retorne True ou False. */ 

let nota1 = prompt("Nota 1: ");
let floatNota1 = parseFloat(nota1);

let nota2 = prompt("Nota 2: ");
let floatNota2 = parseFloat(nota2);

let media = (floatNota1 + floatNota2) / 2;

let aprovado = (media >= 7) ||( media >= 6 && floatNota1 >= 5 && floatNota2 >= 5);

let stringAprovado = `Aluno aprovado? ${aprovado}`;

console.log(stringAprovado);

File: logica-programacao/python/034.py
# Peça duas notas.
# A pessoa é aprovada se:
# média ≥ 7 ou média ≥ 6 E nenhuma nota é menor que 5
# Retorne True ou False.

nota_1 = input("Nota 1: ")
float_nota_1 = float(nota_1)

nota_2 = input("Nota 2: ")
float_nota_2 = float(nota_2)

media = (float_nota_1 + float_nota_2) / 2

aprovado = (media >= 7) or (media >= 6 and float_nota_1 >= 5 and float_nota_2 >= 5)

string_aprovado = f"Aluno aprovado? {aprovado}"

print(string_aprovado)